autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/store/AutofillStore.kt

```diff
@@ -87,6 +87,11 @@ interface AutofillStore {
      */
     suspend fun getAllCredentials(): Flow<List<LoginCredentials>>
 
+    /**
+     * Returns all credentials that have been modified since the given timestamp
+     */
+    suspend fun getAllCredentialsModifiedSince(modifiedSince: Long): List<LoginCredentials>
```

Comment:
Since changing a public API, should be following this right?: [[Doc] Android Public APIs](https://app.asana.com/0/1202561462274611/1205060262189411/f)

Reply:
This is now removed as I'm moving away from using AutofillStore (as discussed)